src/Property.ts

```diff
@@ -212,6 +221,9 @@ export default class Property<T = any> {
 			case this.isString(/in$/):
 				return this.getNumber() * this.getDpi(/* viewPort */);
 
+			case this.isString(/%$/) && isFontSize:
```

Comment:
diff looks prettier but now we're running the `%` regex twice.

Reply:
Do it like you want, by `case` or `if` 😃

Reply:
i'll keep style ... we probably want to use `endsWith` or something in the long run if we want higher performance here over regex